loky/backend/context.py

```diff
@@ -87,40 +82,37 @@ def cpu_count(only_physical_cores=False):
     any other way such as: process affinity, restricting CFS scheduler policy
     or the LOKY_MAX_CPU_COUNT environment variable. If the number of physical
     cores is not found, return the number of logical cores.
- 
+
     It is also always larger or equal to 1.
     """
-    os_cpu_count = os.cpu_count()
+    os_cpu_count = os.cpu_count() or 1
```

Comment:
This is a fix to a known bug? It does not seem related to dropping support for old Python versions.

If this is a fix. If should probably be moved to a dedicated PR with its own changelog entry to document it, and in particular explaining when this can happen.

I supposed it's not possible to write a test for this, but I would love to learn why this is expected to happen.

Reply:
According to docs of [`os.cpu_count`](https://docs.python.org/3/library/os.html#os.cpu_count), it can return either `int` or `None`.
That's why `mypy` (which I use) complains here that `os_cpu_count` is not just `int` but `Union[int, None]`.
In runtime `or 1` branch never happen with latest python, so this change is only to mute linter here.